comparisons/serializers.py

```
@@ -0,0 +1,84 @@
+
+    # [핵심] 생성 시 5개 제한 로직
+    def validate_companies(self, value):
+        if len(value) > 5:
+            raise serializers.ValidationError("기업은 최대 5개까지만 담을 수 있습니다.")
+        return value
+
+    def create(self, validated_data):
+        company_codes = validated_data.pop('companies', [])
+        comparison = Comparison.objects.create(**validated_data)
+
+        unique_codes = list(dict.fromkeys(company_codes))
```

Ruff (0.14.11)

Local variable `unique_codes` is assigned to but never used

Remove assignment to unused variable `unique_codes`

(F841)